spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubEmulatorConfigurationTests.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright 2017 original author or authors.
```

Comment:
It's already far away not `2017`.

We develop here Open Source, more over it is Spring Framework.
Let's indeed make it as the best Framework from all aspects as it stands for a while already ! 😄